### DISCUSSION TITLE
Best practice to deploy/release changes with SSG & SSR on zeit/now

### DISCUSSION THREAD

**Discussion by dohomi** [vercel/next.js#10916] [Help] [answered]
Hi Guys,

I read through the new exciting 9.3 release and I am wondering how I would get these goodies into my current SSR rendered website kit (Storyblok CMS + NextJS). What would be the best practice to get this seamlessly integrated into project - for example to run a `build` task if a CMS admin decides to bring changes online and hit a `ready to release` button somewhere in the headless CMS. I want to continue using zeit/now for my websites so I am curious how that would look like.

Thanks for any hints!

**Comment by timneutkens** [comment, marked as answer]
You can set up a deploy hook: https://zeit.co/docs/v2/more/deploy-hooks
When you change something on the CMS you trigger a rebuild.

**Reply by dohomi** [reply]
Thanks I will look into it. I guess a deploy-hook would simply re-deploy a whole project, so it won't be possible to re-trigger the build of just one certain page? Let's say the user would click `publish` on a certain page inside the CMS that only this single page would be re-build? That would be ideal I think

**Reply by flutterforsale** [reply]
@dohomi I am also looking for a solution https://github.com/zeit/next.js/discussions/10820

**Reply by samuells** [reply]
I know this is a discussed topic through all the frameworks - big topic in Nuxt.js also and I would be really interesting to see it live.

**Comment by flutterforsale** [comment]
+1

**Reply by timneutkens** [reply]
Posting +1 on GitHub is generally considered a bad practice, use emoji reactions.

**Comment by samuells** [comment]
@dohomi you can set multiple different webhooks in Storyblok. It depends also on your plan. So check this link for more info - https://www.storyblok.com/docs/Guides/using-storyblok-webhooks and if you miss something feel free to request it in our missing feature form - https://www.storyblok.com/developers#start - we really care about those request 😏

**Reply by dohomi** [reply]
@samuells thanks for your reply I got everything up and running already. I'll mark this issue as resolved
